nvtabular/utils.py

```diff
@@ -168,3 +168,26 @@ def global_dask_client(client):
         except ValueError:
             return None
     return None
+
+
+class Namespace(SimpleNamespace):
```

Comment:
If I understand correctly, the `SimpleNamespace` class provides syntactic sugar that allows you to create (for example) an object that acts like a namespace to organize member variables so you can do things like `object_from_custom_class.namespace.property` without having to use the dict indexing syntax like `object_from_custom_class.namespace["property"]`.

I think the advice from the Python docs is to use `namedtuple` if what you need is a structured record type though, which is what `DatasetCollection` appears to be. Is there a particular benefit of one over the other here? Are they essentially interchangeable?